sdk/openai/ci.yml

```diff
@@ -42,6 +46,10 @@ parameters:
     displayName: 'azure-ai-openai-assistants'
     type: boolean
     default: true
+  - name: release_azureaiopenairealtime
+    displayName: 'azure-ai-openai-realtime'
+    type: boolean
+    default: true
```

Comment:
Let's set this default to `false` as we don't want to accidentally release this to Maven Central.